slices/bash.yaml

```diff
@@ -1,10 +1,19 @@
 package: bash
 
 slices:
+  config:
+    contents:
+      /etc/bash.bashrc:
+      /etc/skel/.bash_logout:
+      /etc/skel/.bashrc:
+      /etc/skel/.profile:
+
   bins:
     essential:
+      - bash_config
       - libc6_libs
       - libtinfo6_libs
     contents:
-      # user should manually link /bin/sh to /bin/bash
```

Comment:
please keep this comment. My previous remark was wrong and Rafid was on point.

Reply:
Gotcha, added, thanks!